docs/publishers/config.rst

```diff
@@ -450,6 +450,30 @@ loads.
                               // relative from the receiving iframe.
     }
 
+.. option:: sideBySide
+
+  ``Object``. This option lets you customize how the side-by-side mode behaves.
```

Comment:
I think it would be useful to explain briefly what "side-by-side mode" means here.